github/Repository.py

```diff
@@ -1856,10 +1856,10 @@ def get_releases(self):
     def get_release(self, id):
         """
         :calls: `GET /repos/:owner/:repo/releases/:id https://developer.github.com/v3/repos/releases/#get-a-single-release
-        :param id: int (release id), str (tag name)
+        :param id: int (release id), str (tag name), latest (latest release)
```

Comment:
This makes it look like latest is a type.  How about "... str (tag name or "latest")"?